### PR TITLE
replace more logic with canUserPerform

### PR DESCRIPTION
**Pull request by zanderz**
@keybase/react-hackers

### CODE REVIEW

**Comment by zanderz** [comment]
@buoyad I think we are close to removing the last uses of implicitAdmin but I can't quite square them with the new permissions - can you suggest?

**Comment by zanderz** [comment]
looks like the remaining implicitAdmin question is captured in https://keybase.atlassian.net/browse/DESKTOP-5780, so this one's ready for a review

File: shared/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "keybase",
-  "version": "1.0.1",
+  "version": "1.3.2",
```
Comment (cjb) [inline shared/package.json]: What's this change?

File: shared/teams/team/member/container.js [hunk 2]
```diff
@@ -98,7 +96,7 @@ const mergeProps = (stateProps: StateProps, dispatchProps: DispatchProps) => {
   // otherwise you just need to be an admin
   let admin = user.type === 'owner' ? you.type === 'owner' : you.type === 'owner' || you.type === 'admin'
   if (stateProps.teamname.includes('.')) {
-    admin = admin || stateProps._implicitAdminUsernames.contains(you.username || '')
+    admin = admin || stateProps.yourOperations.manageMembers
```
Comment (cjb) [inline shared/teams/team/member/container.js]: It seems like this entire block should be replaced, right? Why are we still doing things like testing owner and admin directly?

File: shared/package.json (hunk 1 above)
Comment (zanderz) [reply shared/package.json]: some merge error, IDK

**Comment by cjb** [comment]
Hi, I took another look:

> shared/package.json

Changes intentional?

> // If they're an owner, you need to be an owner to edit them

Comment seems out of date now.

> let admin = user.type === 'owner' ? you.type === 'owner' : stateProps.yourOperations.manageMembers

Hm, I still don't get it -- why are we continuing to test `user.type` explicitly now that `yourOperations` exists?

**Comment by buoyad** [comment]
@cjb for

> let admin = user.type === 'owner' ? you.type === 'owner' : stateProps.yourOperations.manageMembers

I believe this is because if `user` is an owner, you must be an owner to edit their membership. It's to prevent admins from editing owners' membership.

**Comment by zanderz** [comment]
RE: package.json, I don't know why it changed in the first place, and when I edited it back some whitespace got changed. Could back this out altogether but I thought the whitespace harmless.

RE: testing user, etc: yourOperations just covers your relationship to the team in question, the canUserPerform API doesn't take other individual users into account.

**Comment by chrisnojima** [comment]
lets never check in anything that we don't know why it changed

**Pull request merged**